ssm/hmm.py

```diff
@@ -236,7 +240,9 @@ def sample(self, T, prefix=None, input=None, tag=None, with_noise=True):
         for t in range(pad, pad+T):
             Pt = self.transitions.transition_matrices(data[t-1:t+1], input[t-1:t+1], mask=mask[t-1:t+1], tag=tag)[0]
             z[t] = npr.choice(self.K, p=Pt[z[t-1]])
-            data[t] = self.observations.sample_x(z[t], data[:t], input=input[t], tag=tag, with_noise=with_noise)
+            # TODO - revert this back to how it was
```

Comment:
looks like we don't need this comment anymore?

Reply:
Thanks Scott for catching this!  I just removed this line (and also took the opportunity to update a couple of lines of the plotting code within the jupyter notebook) and created a new commit.  Thanks very much for reviewing all of this!  I really appreciate it.